napari/components/viewer_model.py

```diff
@@ -668,7 +668,11 @@ def _update_active_layer(self, event):
             self.active_layer = active_layer
 
     def _on_layers_change(self, event):
-        self.dims.range = self._calc_layers_ranges()
+        range = self._calc_layers_ranges()
```

Comment:
Don't overwrite the `range` builtin here. `layer_range`?

Reply:
fixed